.github/dependabot.yml

```diff
@@ -3,5 +3,6 @@ updates:
   - package-ecosystem: 'npm'
     directory: '/'
     open-pull-requests-limit: 25
+    versioning-strategy: increase-if-necessary
     schedule:
```

Comment:
add `reviewers` (https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#reviewers)

@DnsZhou will be primary reviewer for this repository